src/vs/workbench/parts/debug/electron-browser/repl.ts

```diff
@@ -230,6 +232,19 @@ export class Repl extends Panel implements IPrivateReplService {
 		this.layout(this.dimension);
 	}
 
```

Comment:
I do not like code duplication, can you please look into removing the `CopyAllAction` from `electronDebugActions.ts`. I believe that we can now always use this action instead, can you please try that?